packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -445,19 +586,30 @@ export class AnchorSet {
 			},
 			onMoveOut: (start: number, count: number, id: Delta.MoveId): void => {
 				assert(parentField !== undefined, 0x3a9 /* Must be in a field to move out */);
+				maybeWithNode(
+					(p) => p.events.emit("childrenChange", p),
+					() => this.events.emit("childrenChange", this),
+				);
 				moveTable.set(id, { parent, parentField, parentIndex: start });
 			},
 			onMoveIn: (start: number, count: number, id: Delta.MoveId): void => {
 				assert(parentField !== undefined, 0x3aa /* Must be in a field to move in */);
+				maybeWithNode(
+					(p) => p.events.emit("childrenChange", p),
+					() => this.events.emit("childrenChange", this),
+				);
 				const srcPath =
 					moveTable.get(id) ?? fail("Must visit a move in after its move out");
 				this.moveChildren(count, srcPath, { parent, parentField, parentIndex: start });
 			},
-			onSetValue: (value: Value): void => {},
+			onSetValue: (value: Value): void => {
+				maybeWithNode((p) => p.events.emit("valueChange", p, value));
```

Comment:
Will it be a pain point that modifying the value of a direct child does not raise 'childrenChange/fieldChange' on the parent?

Reply:
I think this is a good reason not to call the event "fieldChange". Anyway issues like this are things the people working on the actual app facing event API should think about (and they are mostly blocked waiting for this PR). At this level (anchors) that really matters is that there are enough entry points for the event binder to work, not that they have the most intuitive behavior when viewed from the perspective of an app data model.